website/_dogfooding/dogfooding.config.ts

```diff
@@ -90,6 +90,9 @@ export const dogfoodingPluginInstances: PluginConfig[] = [
       id: 'pages-tests',
       path: '_dogfooding/_pages tests',
       routeBasePath: '/tests/pages',
+      showLastUpdateTime: true,
+      showLastUpdateAuthor: true,
+      editUrl: () => 'url placeholder',
```

Comment:
Let's use a real value here so that the dogfood pages can be edited by clicking the edit URL link

this relative URL makes the site build fail